community/OM-Operator-Management/cnv-samples/cnv-operator-install/policy-cnv.yaml

```diff
@@ -0,0 +1,179 @@
+                    deployKubevirtIpamController: false
+                  workloadUpdateStrategy:
+                    batchEvictionInterval: 1m0s
+                    batchEvictionSize: 10
+                    workloadUpdateMethods:
+                      - LiveMigrate
+                  uninstallStrategy: BlockUninstallIfWorkloadsExist
+                  resourceRequirements:
+                    vmiCPUAllocationRatio: 10
+# Fix our KVM_EMULATION setting on the Subscription
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: operators.coreos.com/v1alpha1
+                kind: ClusterServiceVersion
+                metadata:
+                  name: kubevirt-hyperconverged-operator.v4.17.3
+                  namespace: openshift-cnv
+                spec:
+                  install:
+                    spec:  
+                      deployments:
+                        - name: hco-operator
+                          spec:
+                            template:  
+                              spec:
+                                containers:
+                                  - name: hyperconverged-cluster-operator
+                                    env:
+                                    - name: KVM_EMULATION
+                                      value: 'false'
+
+# Hostpath provisioner
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: hostpathprovisioner.kubevirt.io/v1beta1
+                kind: HostPathProvisioner
+                metadata:
+                  name: hostpath-provisioner
+                spec:
+                  imagePullPolicy: IfNotPresent
+                  storagePools:
+                    - name: local
+                      path: /var/hpvolumes
+                      pvcTemplate:
+                        accessModes:
+                          - ReadWriteOnce
+                        resources:
+                          requests:
+                            storage: 50Gi
+                  workload:
+                    nodeSelector:
+                      kubernetes.io/os: linux
+
+#Validate that things are installed
+    - objectDefinition:
+        apiVersion: policy.open-cluster-management.io/v1
+        kind: ConfigurationPolicy
+        metadata:
+          name: kubevirt-hyperconverged-available
+        spec:
+          remediationAction: inform
+          object-templates:
+            - complianceType: musthave
+              objectDefinition:
+                apiVersion: hco.kubevirt.io/v1beta1
+                kind: HyperConverged
+                metadata:
+                  name: kubevirt-hyperconverged
+                  namespace: openshift-cnv
+                status:
+                  conditions:
+                    - message: Reconcile completed successfully
+                      reason: ReconcileCompleted
+                      status: 'True'
+                    - type: Available
```

Comment:
I believe you can include this status in the "spec" policy above - even if it is enforced, ConfigurationPolicy knows not to try to update the status fields, and I believe it will work the way you want (updating the spec, and reporting on the status). But of course, it's not necessary to combine them, and it might be more useful to keep them separate to keep things clear. Just thought it was worth mentioning.